doc/reference/reference_lua/tap.rst

```diff
@@ -309,23 +309,32 @@ so on.
 
     .. _taptest-istype:
 
-    .. method:: isnil(value, test-name)
-                isstring(value, test-name)
-                isnumber(value, test-name)
-                istable(value, test-name)
-                isboolean(value, test-name)
-                isudata(value, test-name)
-                iscdata(value, test-name)
+    .. method:: isnil(test, v, message, extra)
+                isstring(test, v, message, extra)
+                isnumber(test, v, message, extra)
+                istable(test, v, message, extra)
+                isboolean(test, v, message, extra)
+                isudata(test, v, utype, message, extra)
+                iscdata(test, v, ctype, message, extra)
 
         Test whether a value has a particular type. Displays a long message if
         the value is not of the specified type.
 
-        :param lua-value value:
-        :param string test-name: name of the test
+        :param string test: name of the test
+        :param lua-value v: value the type of which is to be checked
+        :param string utype: type of data that a passed value `v` should have
+        :param string ctype: type of data that a passed value `v` should have
+        :param string message: text that will be shown to the user in case of failure
+        :param string extra: additional field
```

Comment:
Probably we can omit parameter `extra`, it looks for me that it is a internal parameter.
See source code https://github.com/tarantool/tarantool/blob/master/src/lua/tap.lua
Maybe @Totktonada can help with it.